Python/optimizer.c

```diff
@@ -539,6 +549,24 @@ translate_bytecode_to_trace(
             oparg = (oparg & 0xffffff00) | executor->vm_data.oparg;
```

Comment:
GitHub won't let me review above this line, but I think the `EXTENDED_ARG` and `ENTER_EXECUTOR` cases immediately above should be swapped now (since `ENTER_EXECUTOR` can sit on top of the first `EXTENDED_ARG`).